qa/suites/fs/cephadm/renamevolume/1-rename.yaml

```diff
@@ -1,7 +1,11 @@
 tasks:
 - cephadm.shell:
     host.a:
+      - ceph fs fail foo
+      - ceph fs set foo refuse_client_session true
       - ceph fs volume rename foo bar --yes-i-really-mean-it
```

Comment:
i know this might be trivial but what about first setting the `refuse_client_session` and then failing the fs? reason being - it sounds safer to fail the fs when it is no more accepting any new connections

Reply:
Doesn't matter since the execution context is this very yaml fragment.